experiments-core/src/foedus/ycsb/ycsb_driver.cpp

```diff
@@ -547,8 +734,8 @@ ErrorStack YcsbDriver::run() {
 
   channel->stop_flag_.store(true);
 
-  // Let's forcibly kill everything if it doesn't stop after 10x the experiment duration.
-  const uint64_t session_timeout_us = FLAGS_duration_micro * 10;
+  // Let's forcibly kill everything if it doesn't stop after 10x the experiment duration + 10 sec.
+  const uint64_t session_timeout_us = FLAGS_duration_micro * 10 + 10000000ULL;
```

Comment:
Ah, this was when we had bugs in the lock. not needed now, but might be good to have.